src/cluster.py

```diff
@@ -43,6 +43,8 @@
 
 PG_BASE_CONF_PATH = f"{POSTGRESQL_CONF_PATH}/postgresql.conf"
 
+RUNNING_STATES = ["running", "streaming"]
```

Comment:
[Patroni 3.0.4](https://patroni.readthedocs.io/en/latest/releases.html#version-3-0-4) exposes the status of replicas using `pg_stat_get_wal_receiver()` so the replicas are usually in streaming state.